pretty_assertions/src/lib.rs

```diff
@@ -430,3 +431,35 @@ macro_rules! assert_matches {
         }
     });
 }
+
+// Not public API. Used by the expansion of this crate's assert macros.
+#[doc(hidden)]
+pub mod private {
+    pub trait CompareAsStrByDefault: AsRef<str> {}
```

Comment:
As pointed out by CI, this needs a
```suggestion
    #[cfg(feature = "alloc")]
    use alloc::string::String;

    pub trait CompareAsStrByDefault: AsRef<str> {}
```
to support no_std environments.

Reply:
Fixed